client/src/components/RelatedObjectsTable.js

```diff
@@ -59,7 +79,13 @@ const RelatedObjectsTable = ({
                         <RemoveButton
```

Comment:
I was curious about this `id` assigned to `td` at line 78. It seems like we don't use it anywhere,which means increasing build bundle size for no reason.

Reply:
good question, but that change was introduced in a different PR, so off-topic

Reply:
@cemalettin-simsoft The intention was to use it in the wdio tests, but in the end I decided there just to use:
```
  getTestMultiReferenceFieldValueRow(n) {
    return this.testMultiReferenceFieldValue.$(`tbody tr:nth-child(${n})`)
  }
```
and do:
```
      // Delete one of the selected values
      CreateReport.getTestMultiReferenceFieldValueRow(3).$("button").click()
```
So the `id` is indeed not really necessary; it can either stay or go, es ist mir egal, but as @VassilIordanov says: unrelated to this PR.

Reply:
Yeah, i know it is unrelated but opening a new pr is just a big overhead for simple but useful changes like this. Which makes small mistakes/errors/leftovers accumulate over time since they are, most of the time, unrelated to the big issue.